mlrun/datastore/base.py

```diff
@@ -261,7 +261,7 @@ def reader(*args, **kwargs):
                             updated_args = [f"{base_path}/{filename}"]
                             updated_args.extend(args[1:])
                             dfs.append(df_module.read_csv(*updated_args, **kwargs))
-                        return pd.concat(dfs)
```

Comment:
Is this a bug fix?

Reply:
yes, it changed to `df_module.concat(dfs)`.

Reply:
Consider opening a ticket for it. And add a regression test.

Reply:
There is a regression test.
That is why I added a test with dask dataframe.